tests/linearizable/ctest/breakloop.c

```diff
@@ -0,0 +1,308 @@
+#include <alloca.h>
```

Comment:
Run nemesis in a loop.  This utility was created to study a cluster's recovery time after a partition is healed.